_templates/modules/component/component.unit.spec.ts.ejs

```diff
@@ -4,15 +4,15 @@ to: src/js/components/<%= h.inflection.pluralize(h.changeCase.lower(type)) %>/<%
 import 'babel-polyfill';
 import { build } from 'vuenit';
 
-import { data } from '../data/<%= h.inflection.pluralize(h.changeCase.lower(type)) %>.data.ts';
+import { data } from './data/<%= h.changeCase.paramCase(name) %>data';
```

Comment:
think this should be:
`import { data } from './data/<%= h.changeCase.paramCase(name) %>.data';`